src/backend/bacon-video-widget-gst-missing-plugins.c

```diff
@@ -59,53 +59,6 @@ typedef struct
 }
 TotemCodecInstallContext;
 
-#ifdef GDK_WINDOWING_X11
```

Comment:
I generally agree, responses inline:

> what's the downside of keeping the PK integration code? We don't ship PK, so as long as the missing daemon is not seen as an error, we should still be OK; otherwise maybe we can fix this upstream.

We need to bypass the PK integration since we don't ship PK. The old code attempted to look up properties on the dbus object which would fail, so you would never see the dialog. I'm in favour of removing the code in a separate commit

> As for the gst helper code, rather than removing it, I'd prefer to keep it there even if unused, so to minimize changes compared do downstream. So a first commit would simply make sure the current integration is disabled/not running.

I think that's fine

> in a separate commit, you can add the MRL getter (hopefully we can bring that upstream too)

Yup, makes sense